src/verifyUtils.ts

```diff
@@ -429,26 +410,26 @@ export async function verifyProofElement(
 
         /* all is good, no throw */
     }
-    if (proof.type === 'CordSDRProof2024') {
+    if (proof.type === 'CordSDRProof2025') {
```

Comment:
same as below. lets make sure the existing proof also gets verified if present.

(move it to another file may be as backward-compatibility.ts may be?)